c7n/tags.py

```diff
@@ -369,6 +369,10 @@ class Tag(Action):
     batch_size = 25
     concurrency = 2
 
+    deprecations = (
+        deprecated.alias('mark'),
```

Comment:
for the deprecation with alias, do we render with the preferred/current name as output?

Reply:
From the test output below:
```
policy 'dep-test'
  actions:
    remove-tag: alias 'unmark' has been deprecated
  ```
So I think so. The current name is rendered first, followed by the deprecated alias.